cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1,6 +1,7 @@
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
 **Table of Contents**
 
+- [Creating a safe space](#creating-a-safe-space)
```

Comment:
The items in the playbook aren't currently listed in order of importance.

I don't think it's premature optimization to link the section from the README because how else will the leaders discover the section?